build_stage_repository.py

```diff
@@ -0,0 +1,182 @@
+#!/usr/bin/env python3
+
+from subprocess import check_output, STDOUT, CalledProcessError
+import os
+import yaml
+import glob
+import shutil
+import time
+import hashlib
+import sys
+
+
+def execute(command):
+    try:
+        return check_output(command, universal_newlines=True, stderr=STDOUT)
+    except CalledProcessError as error:
+        print(error.output)
+        return False
```

Comment:
For type checking, would it be better if the exception was actually just raised? Right now I don't know if you want to differentiate between success but no output and False.